fulcio.swagger.json

```diff
@@ -114,12 +114,10 @@
       "type": "object",
       "properties": {
         "credentials": {
-          "$ref": "#/definitions/v2Credentials",
-          "title": "Identity information about who possesses the private / public key pair presented"
```

Comment:
@bobcallaway fyi

Hm, this is removing the description and title, that's wrong

Reply:
looks like it was this change: https://github.com/grpc-ecosystem/grpc-gateway/pull/2904/files#diff-0330d83e3109c5c0a8cb78a2a980db731e973c738da7851d34a88e00244a4b5e

Reply:
should we hold this?

Reply:
yes, probably should

Reply:
Fixed it! Needed to be recreated